src/github.com/matrix-org/go-neb/services/github/github.go

```diff
@@ -21,6 +23,31 @@ import (
 // E.g. owner/repo#11 (issue/PR numbers) - Captured groups for owner/repo/number
 var ownerRepoIssueRegex = regexp.MustCompile(`(([A-z0-9-_]+)/([A-z0-9-_]+))?#([0-9]+)`)
 var ownerRepoRegex = regexp.MustCompile(`^([A-z0-9-_]+)/([A-z0-9-_]+)$`)
+var issueListTemplate = template.Must(template.New("issueList").Parse(`
+{{ if .Issues }}
+	<ul>
+		{{ range .Issues }}
+			<li>
+				<a href="{{ .HTMLURL }}">
+					<strong>#{{ .Number }}</strong>
+					{{ .Title }}
+					<strong>({{ .State }})</strong>
+					{{ range .Labels }}
+						<a href="{{ .URL }}">
```

Comment:
This is the github API URL and not HTML. Clicking this link won't work as you'd expect I think (api.github.com returns JSON)